src/app/ConfigureUtilities.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import IconSettings from 'terra-icon/lib/icon/IconSettings';
+
+function generateItemConfig(defaultItem, items, key) {
+  const childKeys = {};
+  items.forEach((item) => {
+    childKeys[item] = {
+      key: item,
+      title: item,
+      isSelected: defaultItem === item,
+      isSelectable: true,
+    };
+  });
+
+  return {
+    key,
+    title: `${key}: ${defaultItem}`,
+    childKeys,
+  };
+}
+
+function convertMenuItemToArray(menuItem) {
+  const updatedItem = Object.assign({}, menuItem);
+  let menuItems = [];
+  if (updatedItem.childKeys) {
+    menuItems = menuItems.concat(Object.keys(updatedItem.childKeys).reduce((acc, item) => acc.concat(convertMenuItemToArray(updatedItem.childKeys[item])), []));
+    updatedItem.childKeys = Object.keys(updatedItem.childKeys);
+  }
+  menuItems.push(updatedItem);
+  return menuItems;
+}
+
+function updateMetaData(parentMenuItem, metaData) {
+  if (parentMenuItem && metaData) {
+    Object.keys(parentMenuItem.childKeys).forEach((item) => {
+      // eslint-disable-next-line no-param-reassign
+      parentMenuItem.childKeys[item].metaData = metaData;
+    });
+  }
+}
+
+function updateSelectedItem(parentMenuItem, selectedItem) {
+  if (parentMenuItem && selectedItem) {
+    // eslint-disable-next-line no-param-reassign
+    parentMenuItem.title = `${parentMenuItem.key}: ${selectedItem}`;
+    Object.keys(parentMenuItem.childKeys).forEach((item) => {
+      // eslint-disable-next-line no-param-reassign
+      parentMenuItem.childKeys[item].isSelected = item === selectedItem;
+    });
+  }
+}
+
+class ConfigureUtilties {
+
+  static generateInitialUtiltiesConfig(appConfig) {
```

Comment:
you should be able to move these static functions to use the arrow syntax above and then to improve readability on what functions are exported, we could update to be like:
```
const ConfigureUtilties = {
    generateInitialUtiltiesConfig,
    convertChildkeysToArray,
   ... etc
};

export default ConfigureUtilties;
```

Reply:
I prefer the class syntax.